tests/Midi_gtest.cpp

```diff
@@ -638,5 +638,40 @@ TEST_F(MidiTest, badData)
         msgs[1] = i;
         Parse(msgs, 2);
     }
+    EXPECT_FALSE(midi.HasEvents());
+}
+
+// send some status bytes without data between valid messages with running status
+TEST_F(MidiTest, runningStatusAndStatusBytes)
+{
+    uint8_t noteon            = 0x90;
+    uint8_t running_status[6] = {0x40, 100, 0x40, 0x01, 0x40, 100};
+    uint8_t status_bytes[3]   = {0x90, 0x91, 0xB3};
+
+    Parse(&noteon, 1);
+    for(int i = 0; i < 3; i++)
+    {
+        MidiEvent   event   = ParseAndPop(&running_status[i * 2], 2);
+        NoteOnEvent noEvent = event.AsNoteOn();
```

Comment:
`noEvent` sounds very much like "no event". Would `noteOnEvent` or `onEvent` make it easier to read?